### PR TITLE
Run proofreading over the whole library

### PR DESCRIPTION
**Pull request by kolibril13**
I ran a spell checker through the whole library and corrected lots of words in the docstrings.
Also, there were some spelling mistakes in some variables, which I fixed as well.
When this happened, I used the refactor tool of pycharm to make sure that all other variable names are changed as well.

During the process, I added manim specific words to the dictionary, that were not recognized by the default spell checker.
Here they are:
```console
anims
bézier
centerpoint
circ
dvisvgm
infty
lerp
manim
manim's
manimce
mobject
mobject's
mobjects
ndarray
pointwise
rgbas
rgbs
subfolder
submob
submobject
submobjects
submobs
subpaths
tipable
unpythonic
vect
vects
vmob
vmobject
vmobjects
```

```
-  FIx Spelling in docstrings and variables(:pr:`PR NUMBER HERE`)
```

## Acknowledgements
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by naveen521kk** [comment]
I would like to have a test on that for each PR. What tool you used and how to run it in CI? I will add a check for that here.

**Comment by kolibril13** [comment]
> I would like to have a test on that for each PR. What tool you used and how to run it in CI? I will add a check for that here.

Good idea, I used the pycharm internal spell-checker following this instruction: https://stackoverflow.com/a/37866721/6933377

**Comment by naveen521kk** [comment]
> Good idea, I used the pycharm internal spell-checker following this instruction: https://stackoverflow.com/a/37866721/6933377

There should be something which can be run from CI. We should find something.

File: manim/mobject/geometry.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def add_tip(self, tip=None, tip_shape=None, tip_length=None, at_start=False):
 
     def create_tip(self, tip_shape=None, tip_length=None, at_start=False):
         """
-        Stylises the tip, positions it spacially, and returns
+        Stylises the tip, positions it specially, and returns
```
Comment (behackl) [inline manim/mobject/geometry.py:137]:
```suggestion
        Stylises the tip, positions it spatially, and returns
```

or

```suggestion
        Stylises the tip, positions it, and returns
```

File: manim/mobject/svg/text_mobject.py [hunk 2]
```diff
@@ -477,7 +477,7 @@ class Paragraph(VGroup):
     Parameters
     ----------
     line_spacing : :class:`int`, optional
-        Represents the spaning betweeb lines. Default to -1, which means auto.
+        Represents the spanning between lines. Default to -1, which means auto.
```
Comment (behackl) [inline manim/mobject/svg/text_mobject.py:480]:
```suggestion
        Represents the spacing between lines. Default to -1, which means auto.
```

**Review by behackl — APPROVED** [review]
LGTM in general, apart from black failing. Found two wrong corrections.

**Review by naveen521kk — APPROVED** [review]

**Comment by kolibril13** [comment]
Thanks for the quick review!

File: example_scenes/basic.py [hunk 3]
```diff
@@ -29,7 +29,7 @@ def construct(self):
         transform_title.to_corner(UP + LEFT)
         self.play(
             Transform(title, transform_title),
-            LaggedStart(*map(FadeOutAndShiftDown, basel)),
+            LaggedStart(*map(lambda obj: FadeOutAndShift(obj, direction=DOWN), basel)),
```
Comment (behackl) [inline example_scenes/basic.py:32]: While we are at it, we could also remove the map here:

```suggestion
            LaggedStart(*[FadeOutAndShift(obj, direction=DOWN) for obj in basel]),
```

**Comment by kolibril13** [comment]
> > Good idea, I used the pycharm internal spell-checker following this instruction: https://stackoverflow.com/a/37866721/6933377
> 
> There should be something which can be run from CI. We should find something.

Yes, I found this one:
https://pypi.org/project/flake8-spellcheck/

I will merge this in a few hours before I will make the strings in the manim opening scene a bit prettier.

**Pull request merged**